applications/nrf_desktop/doc/power_manager.rst

```diff
@@ -1,7 +1,132 @@
+| :ref:`ble_state`                 | ``ble_peer_event``     |                                  |                      |                                    |
++----------------------------------+------------------------+                                  +----------------------+------------------------------------+
+| :ref:`usb_state`                 | ``usb_state_event``    |                                  |                      |                                    |
++----------------------------------+------------------------+                                  +----------------------+------------------------------------+
+| :ref:`hid_state`                 | ``hid_report_event``   |                                  |                      |                                    |
++----------------------------------+------------------------+                                  +----------------------+------------------------------------+
+| :ref:`nrf_desktop_power_manager` | ``power_down_event``   |                                  |                      |                                    |
++----------------------------------+------------------------+                                  +----------------------+------------------------------------+
+|                                  | ``wake_up_event``      |                                  |                      |                                    |
++----------------------------------+------------------------+----------------------------------+----------------------+------------------------------------+
+
+
+External dependencies
+*********************
+
+This module utilizes Zephyr :ref:`zephyr:power_management`.
+It depends on :option:`HAS_SYS_POWER_STATE_DEEP_SLEEP_1` being enabled and selects :option:`DEVICE_POWER_MANAGEMENT` and :option:`SYS_POWER_DEEP_SLEEP_STATES`.
+
+
+Configuration
+*************
+
+The module is enabled by the ``CONFIG_DESKTOP_POWER_MANAGER_ENABLE`` option.
+
+With ``CONFIG_DESKTOP_POWER_MANAGER_TIMEOUT`` configuration option you can set time after which the application will go to low power mode.
+The ``CONFIG_DESKTOP_POWER_MANAGER_STAY_ON`` let system stay idle also when there are no active connections.
+The ``CONFIG_DESKTOP_POWER_MANAGER_ERROR_TIMEOUT`` set the time after which the device is turned off upon an internal error.
+
+
+Implementation details
+**********************
+
+The :ref:`nrf_desktop_power_manager` is started when the :ref:`nrf_desktop_main` module is ready (reported using ``module_state_event``).
+
+The :ref:`nrf_desktop_power_manager` takes control of the overall system power state.
+Except when entering a deep sleep mode the system will stay in ``SYS_POWER_STATE_ACTIVE`` state.
+When entering the deep sleep mode the module will set system to ``SYS_POWER_STATE_DEEP_SLEEP_1`` state.
+
+
+Power down
+==========
+
+If the device is not connected to the USB, the module counts time since the last user interaction (last HID report sent from the device).
```

Comment:
Can you add an introductory sentence that states why the power down is dependant on the USB connection, and also introduces the two options below?

Reply:
Also, can you add an introductory sentence about what is discussed in this section?

Maybe by also redo the whole section into unordered steps or provide an overview at the start of what is going on during power down:
- checking USB
- setting the low power mode
- checking BLE
- reporting low power